src/hiero_sdk_python/tokens/token_nft_transfer.py

```
@@ -5,31 +5,32 @@
     receiver, serial number of the NFT, and whether the transfer is approved.
     """
 
-
     def __init__(
         self,
         token_id: TokenId,
         sender_id: AccountId,
         receiver_id: AccountId,
         serial_number: int,
-        is_approved: bool = False
+        is_approved: bool = False,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)